src/Microsoft.Data.SqlClient/tests/ManualTests/AlwaysEncrypted/ApiShould.cs

```diff
@@ -1812,52 +1812,6 @@ public void TestSqlCommandSequentialAccessCodePaths(string connection, CommandBe
 
         }
 
-        [SkipOnTargetFramework(TargetFrameworkMonikers.Netcoreapp)]
-        [ConditionalTheory(typeof(DataTestUtility), nameof(DataTestUtility.AreConnStringSetupForAE))]
-        [ClassData(typeof(AEConnectionStringProvider))]
-        public void TestExecuteXmlReader(string connection)
```

Comment:
Why are we removing this test?

Reply:
Test has been declared as false test. "FOR XML" clause is not supported in AE.

Reply:
It's okay to test for unsupported scenarios. Is the test not passing consistently?

Reply:
Good point David. I can actually modify it to check for the error message that the mentioned clause is not supported on AE.

Reply:
reverted.